nixos/modules/tasks/filesystems.nix

```diff
@@ -134,6 +134,22 @@ in
 
   config = {
 
+    assertions = [
+      { assertion =
+          builtins.any (fs: [] != intersectLists (splitString "," fs.options)
```

Comment:
Is this something new on master that's not on release-15.09?  I can't
access any of these three without the `builtins` prefix.